### PR TITLE
testing/ruuvi-prometheus: new aport

### PR DESCRIPTION
**Pull request by joneskoo**
ruuvi-prometheus is a Prometheus exporter that receives Bluetooth beacon advertisements from Ruuvi tag sensors and exports them with a Prometheus metrics client (a http server).

### CODE REVIEW

File: testing/ruuvi-prometheus/alpine/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Contributor: Joonas Kuorilehto
+# Maintainer:
```
Comment (maxice8) [inline testing/ruuvi-prometheus/alpine/APKBUILD]: Missing maintainer

File: testing/ruuvi-prometheus/alpine/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+pkgname=ruuvi-prometheus
+pkgver=0.1
+pkgrel=0
+pkgdesc="Ruuvi sensor Prometheus exporter"
+url="https://github.com/joneskoo/ruuvi-prometheus"
+arch="all"
+depends="bluez"
+license="BSD-2-Clause"
+options="!check"  # No test suite
+depends=""
```
Comment (maxice8) [inline testing/ruuvi-prometheus/alpine/APKBUILD]: empty variables can be removed for brevity

File: testing/ruuvi-prometheus/alpine/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+makedepends="go"
+install=""
+subpackages="$pkgname-openrc"
+
+source="$pkgname-$pkgver.tar.gz::https://github.com/joneskoo/ruuvi-prometheus/archive/v$pkgver.tar.gz
+        ruuvi-prometheus.initd
```
Comment (maxice8) [inline testing/ruuvi-prometheus/alpine/APKBUILD]: Use 1 tab indent

File: testing/ruuvi-prometheus/alpine/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        "
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
```
Comment (maxice8) [inline testing/ruuvi-prometheus/alpine/APKBUILD]: not required, this is done automatically

File: testing/ruuvi-prometheus/alpine/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+options="!check"  # No test suite
+depends=""
+makedepends="go"
+install=""
+subpackages="$pkgname-openrc"
+
+source="$pkgname-$pkgver.tar.gz::https://github.com/joneskoo/ruuvi-prometheus/archive/v$pkgver.tar.gz
+        ruuvi-prometheus.initd
+        "
+builddir="$srcdir/$pkgname-$pkgver"
```
Comment (maxice8) [inline testing/ruuvi-prometheus/alpine/APKBUILD]: default value is being used, this line can be removed

**Comment by Ikke** [comment]
Output of apkbuild-lint:

```
IC:[AL7]:APKBUILD:18:indent with tabs
IC:[AL7]:APKBUILD:19:indent with tabs
MC:[AL1]:APKBUILD:20:builddir can be removed as it is the default value
MC:[AL5]:APKBUILD:12:variable set to empty string: depends=""
MC:[AL5]:APKBUILD:14:variable set to empty string: install=""
MP:[AL13]:APKBUILD:23:cd "$builddir" can be removed in phase 'build'
MP:[AL13]:APKBUILD:28:cd "$builddir" can be removed in phase 'package'
```

**Comment by Ikke** [comment]
> testing/ruuvi-prometheus/alpine/APKBUILD

The APKBUILD file (and others) should live in the root of the package dir, not in `alpine`

File: testing/ruuvi-prometheus/alpine/APKBUILD (hunk 4 above)
Comment (joneskoo) [reply testing/ruuvi-prometheus/alpine/APKBUILD]: I think this was from the template generator I used following the docs. I didn't add it. Will clean.

**Pull request closed without merging**